examples/xml-attributes/xml_attributes.bal

```diff
@@ -28,14 +28,14 @@ public function main() {
     // It is possible to get all the attributes at once. However, this cannot be assigned to any variable.
     io:println(x1@);
 
-    // We can use x1@ syntax to get all attributes of a singleton xml element as a map.
+    // We can use `x1@` syntax to get all attributes of a single XML element as a map.
```

Comment:
I think we can leave singleton as is, since this will not work for sequences as described below.